src/index.js

```diff
@@ -152,12 +152,24 @@ var VERSION = process.env.npm_package_version;
  * @param {external:ApplePayPaymentRequest} paymentRequest The payment request details to apply on top of those from Braintree.
  */
 
+/** @typedef {object} googlePayCreateOptions The configuration options for Google Pay. Additional options from the few listed here are available, many have default values applied based on the settings found in the Braintree Gateway. For more information, see [Google's Documentation](https://developers.google.com/pay/api/web/object-reference#request-objects).
+ *
+ * @param {string} merchantId The id provided by Google for processing transactions in production. Not necessary for testing in sandbox.
```

Comment:
s/id/ID

If it is not necessary in sandbox would we want this to be optional in JSDoc?

Reply:
For JSDoc, I think keeping that as required makes sense, as the docs generally refer to the production environment.